pkg/integration/tests/ui/keybinding_suggestions_when_switching_repos.go

```diff
@@ -0,0 +1,45 @@
+package ui
+
+import (
+	"path/filepath"
+
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+var KeybindingSuggestionsWhenSwitchingRepos = NewIntegrationTest(NewIntegrationTestArgs{
+	Description:  "Show correct keybinding suggestions after switching between repos",
+	ExtraCmdArgs: []string{},
+	Skip:         false,
+	SetupConfig: func(config *config.AppConfig) {
+		otherRepo, _ := filepath.Abs("../other")
+		config.AppState.RecentRepos = []string{otherRepo}
+	},
+	SetupRepo: func(shell *Shell) {
+		shell.CloneNonBare("other")
+	},
+	Run: func(t *TestDriver, keys config.KeybindingConfig) {
+		switchToRepo := func(repo string) {
+			t.GlobalPress(keys.Universal.OpenRecentRepos)
+			t.ExpectPopup().Menu().Title(Equals("Recent repositories")).
+				Lines(
+					Contains(repo).IsSelected(),
+					Contains("Cancel"),
+				).Confirm()
+			t.Views().Status().Content(Contains(repo + " → master"))
+		}
+
+		t.Views().Files().Focus()
+		t.Views().Options().Content(
+			Equals("Commit: c | Stash: s | Reset: D | Keybindings: ? | Cancel: <esc>"))
```

Comment:
It's a little awkward that we depend on the concrete keybinding suggestions here, so we'll have to adapt the test when those change. It doesn't bother me too much though, it's still better than not having a test at all.

I did choose the panel that has the lowest number of keybindings.